Src/Support/Google.Apis.Auth.AspNetCore.IntegrationTests/Controllers/AccountController.cs

```diff
@@ -0,0 +1,30 @@
+﻿/*
+Copyright 2018 Google Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Google.Apis.Auth.AspNetCore.IntegrationTests.Controllers
+{
+    public class AccountController : Controller
```

Comment:
In the older AP.NET (not core) MVC package we do provide a controller to assist with auth redirects.
This is not necessary in Core because we're now using the redirect handling built in to the ASP.NET Core auth framework.